src/tests/regenerate_unaudited.rs

```diff
@@ -1,36 +1,36 @@
 use super::*;
```

Comment:
This file should probably be renamed to `regenerate_exemptions.rs`

Reply:
yeah wanted to defer that